OrbitGl/Batcher.cpp

```diff
@@ -102,9 +103,11 @@ void Batcher::AddShadedBox(Vec2 pos, Vec2 size, float z, const Color& color,
 
 void Batcher::AddBox(const Box& box, const std::array<Color, 4>& colors, const Color& picking_color,
                      std::unique_ptr<PickingUserData> user_data) {
-  box_buffer_.boxes_.push_back(box);
-  box_buffer_.colors_.push_back(colors);
-  box_buffer_.picking_colors_.push_back_n(picking_color, 4);
+  float layer = box.vertices[0][2];
```

Comment:
There's an assumption here that all vertices have the same z value, which is indeed true in our case. It might be good to add a comment somewhere about this assumption. (In the Batcher.h header file?)

Reply:
I'm thinking in making another PR changing the way we construct the Triangles, WDYT?